src/osd/OSDMap.cc

```diff
@@ -5796,7 +5796,13 @@ void OSDMap::check_health(CephContext *cct,
   }
 
   // OSD_UPGRADE_FINISHED
-  // none of these (yet) since we don't run until luminous upgrade is done.
+  if (auto require_release = pending_require_osd_release()) {
+    ostringstream ss;
+    ss << "all OSDs are running " << *require_release << " or later but"
+       << " require_osd_release < " << *require_release;
+    auto& d = checks->add("OSD_UPGRADE_FINISHED", HEALTH_WARN, ss.str(), 0);
```

Comment:
https://jenkins.ceph.com/job/ceph-pull-requests/86972/consoleFull#-10065859352a811ea2-3e7b-466b-84b4-d13df7e35809

Reply:
This was a subtle one! checks->add() doesn't take the 'count' parameter in nautilus. Fixed.